### PR TITLE
add feature for container support

### PR DESCRIPTION
**Pull request by Eskibear**

### CODE REVIEW

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/pom.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+          <!-- copy the jar dependencies -->
+          <execution>
+            <id>copy</id>
+            <phase>initialize</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <outputDirectory>${project.build.directory}/lib</outputDirectory>
+              <!--<includeGroupIds>com.spotify,org.apache,org.slf4j,org.glassfish,javax.annotation,javax.ws.rs,com.fasterxml.jackson,com.github.jnr,org.ow2.asm,org.bouncycastle</includeGroupIds>-->
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/pom.xml]: delete this

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/pom.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                <artifactItem>
+                  <groupId>org.bouncycastle</groupId>
+                  <artifactId>bcpkix-jdk15on</artifactId>
+                </artifactItem>
+                <artifactItem>
+                  <groupId>org.apache.commons</groupId>
+                  <artifactId>commons-compress</artifactId>
+                </artifactItem>
+              </artifactItems>
+              <!--<stripVersion>true</stripVersion>-->
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/pom.xml]: delete unused line

**Comment by Eskibear** [comment]
retest please

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/Constant.java [hunk 3]
```diff
@@ -0,0 +1,12 @@
+package com.microsoft.azuretools.container;
```
Comment (adashen) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/Constant.java]: license

**Comment by Eskibear** [comment]
retest this please

**Comment by Eskibear** [comment]
retest this please

**Comment by Eskibear** [comment]
retest this please

**Comment by Eskibear** [comment]
retest this please

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerRunHandler.java [hunk 4]
```diff
@@ -0,0 +1,155 @@
...
+		        if (imageId != null) {
+		          imageIdFromMessage.set(imageId);
+		        }
+		      }
+		    });
+		return imageName;
+	}
+
+	private void export(IProject project, String destinationPath) throws Exception {
+		String projectName = project.getName();
```
Comment (jdneo) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerRunHandler.java]: indent

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerizeHandler.java [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+
+import com.microsoft.azuretools.container.Constant;
+import com.microsoft.azuretools.core.utils.AzureAbstractHandler;
+import com.microsoft.azuretools.core.utils.PluginUtil;
+
+public class DockerizeHandler extends AzureAbstractHandler {
+
+	@Override
+	public Object onExecute(ExecutionEvent event) throws ExecutionException {
+		IWorkbenchWindow window = PlatformUI.getWorkbench().getActiveWorkbenchWindow();
```
Comment (jdneo) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerizeHandler.java]: will window be null?

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/Constant.java (hunk 3 above)
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/Constant.java]: done

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerizeHandler.java (hunk 5 above)
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerizeHandler.java]: I followed code in webapp.  Will change it as you say.

File: PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerRunHandler.java (hunk 4 above)
Comment (Eskibear) [inline PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.container/src/com/microsoft/azuretools/container/handlers/DockerRunHandler.java]: will eliminate tabs

**Pull request merged**